### PR TITLE
Add new class ArrowAssociation

### PR DESCRIPTION
**Pull request by SemBen**

### CODE REVIEW

**Review by sinerger — CHANGES_REQUESTED** [review]
OK

File: UML Diagram drawer/ArrowAssociation.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        {
+            Graphics = graphics;
+            Color = color;
+            Width = width;
+            Pen = new Pen(Color, Width);
+        }
+
+        public override void Draw()
+        {
+            DrawStraightBrokenLine();
```
Comment (kovalenko-Max) [inline UML Diagram drawer/ArrowAssociation.cs]: Add null checks

File: UML Diagram drawer/ArrowAssociation.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            DrawArrowheadAssociation();
+        }
+
+        private void DrawArrowheadAssociation()
+        {
+            Point[] points;
+
+            if (IsHorizontal)
```
Comment (kovalenko-Max) [inline UML Diagram drawer/ArrowAssociation.cs:33]: Add null checks

**Review by kovalenko-Max — APPROVED** [review]

**Review by sinerger — APPROVED** [review]
Ok

**Pull request merged**